Vagrantfile

```diff
@@ -71,7 +71,7 @@ $provision_script = <<SCRIPT
 set -x
 set -e
 ln -nsf /srv/zulip ~/zulip
-/usr/bin/python /srv/zulip/tools/provision.py
+/usr/bin/python /srv/zulip/tools/provision.py | sudo tee -a /var/log/zulip_provision.log
```

Comment:
1. I think it doesn't. Tee is by default on "clean" linux.
2. Added
3. `mkdir abc/non-existing/non-existing | tee log.log`    -> passed 1
Is it an answer?
@timabbott